python/OpenVINO_EP/yolov4_object_detection/README.md

```diff
@@ -36,7 +36,7 @@ The source code for this sample is available [here](https://github.com/microsoft
 * use any sample video with objects as test input to this sample
 * Download the tinyYOLOv2 model from the ONNX Model Zoo
 [Download Sample videos](https://github.com/intel-iot-devkit/sample-videos)
-* For all the python package dependencies requirements, check 'requirements.txt' file in the sample      directory.
+Note: For all the python package dependencies requirements, check 'requirements.txt' file in the sample directory.
```

Comment:
Add: You may install these with pip install -r requirements.txt.

Reply:
Done